raml-200-tutorial.html

```diff
@@ -453,6 +453,8 @@ <h2 id="parameters">PARAMETERS</h2>
         200:
           body:
             application/json:
+              example: |
+                &lt;&lt;exampleCollection&gt;&gt;
```

Comment:
I would put the example on the same line here. i.e. `example: &lt;&lt;exampleCollection&gt;&gt;`